troposphere/static/js/actions/instance/stop.js

```diff
@@ -1,58 +1,88 @@
+import Backbone from "backbone";
 
 import InstanceConstants from "constants/InstanceConstants";
+import Instance from "models/Instance";
 import InstanceState from "models/InstanceState";
-import Utils from "../Utils";
 import InstanceActionRequest from "models/InstanceActionRequest";
 
-export default {
+import Utils from "../Utils";
 
-    stop: function(params) {
-        if (!params.instance)
-            throw new Error("Missing instance");
-
-        var instance = params.instance,
-            instanceState = new InstanceState({
-                status_raw: "active - powering-off",
-                status: "active",
-                activity: "powering-off"
-            }),
-            originalState = instance.get("state"),
-            actionRequest = new InstanceActionRequest({
-                instance: instance
-            });
+/**
+ * Defines the `stop` steps for a single instance
+ */
+function handleOne(instance){
+    // if we've been passed a model via "selected resources" that
+    // is *not* an Instance, then just skip it - _move on_
+    if (!(instance instanceof Instance)) return;
```

Comment:
The reason here is that "selectedResources" will include anything selected, so Instance(s), Volume(s), Image(s), ExternalLink(s). I think tolerating them and skipping over them was better then halting the entire processing with an exception. 

The whole hope here was to "execute" a best effort on what the community member is requesting. If they asked to suspend a volume - just ignore that (etc).